app/assets/v2/css/help.css

```diff
@@ -0,0 +1,79 @@
+body {
+  font-family: "OpenSans";
```

Comment:
we need to consolidate either around Futura or OpenSans... cc @pixelantdesign

Reply:
refer why the change is being made : https://github.com/gitcoinco/web/issues/133#issuecomment-352750331

I couldn't find the conversation which algae12 refers on slack ( due to the free versions memory limited 😓  )